lib/bodeco_module_helper/vagrant.rb

```diff
@@ -7,12 +7,15 @@ def vm(opt)
   url = opt.fetch(:url, '').to_s
   os_type = opt[:os_type] || opt[:type] || :linux
   gui = opt.fetch(:gui, false)
-  port = opt.fetch(:port, nil)
+  ports = Array(opt.fetch(:port, []))
```

Comment:
It would be better, but it would also make it more complicated to use and your not guaranteed to get the port on the host.